ISLE/define.h

```diff
@@ -8,7 +8,7 @@ class Isle;
 extern Isle *g_isle;
 extern int g_closed;
 #define WNDCLASS_NAME "Lego Island MainNoM App"
-extern const char *WINDOW_TITLE;
+#define WINDOW_TITLE "LEGO\xAE"
```

Comment:
Could you move the offset comment to here instead, just for the sake of documentation?

Reply:
Yep, added them for both macros